linera-sdk/wit/contract-system-api.wit

```diff
@@ -127,6 +128,13 @@ interface contract-system-api {
         storage-size-delta: u32,
     }
 
+    variant round {
+        fast,
```

Comment:
If we're in a fast round, should we always fail this oracle call?

Reply:
We kind of do, since oracles are always disabled in fast rounds.

Du you think we should not even expose the variant to contracts? I.e. have a different `Round` type in the SDK with one less variant?

Reply:
I like it! That also solves the edge case with the timeout: Whenever we do have to wait for a timeout, it can't be a multi-leader round anyway.

Reply:
Done in https://github.com/linera-io/linera-protocol/pull/3175/commits/5361eb2a276465a9b019d2db656a7b71083eac62.